angular/src/app/modules/not-have-access/not-have-access.module.ts

```diff
@@ -0,0 +1,25 @@
+import { SharedModule } from './../../../shared/shared.module';
+
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+
+import { TooltipModule } from 'ngx-bootstrap/tooltip';
+import { NotHaveAccessRoutingModule } from './not-have-access-routing.module';
```

Comment:
tạo ra page not-have-access và page payslip-detail-preview-link thôi, tạo ra module làm gì cho cồng kềnh